include/vl53_main.h

```diff
@@ -0,0 +1,35 @@
+/************************************
+ *        VL53 Main header file     *
+ ************************************/
+
+#ifndef __VL53_MAIN_H
+#define __VL53_MAIN_H
+
+// !!! Uncomment the line below to enter debug mode !!!
+#define debug
+
+// Includes
+#include "VL53L1X_api.h"
+#include "VL53L1X_calibration.h"
+
+// Functions
+int VL53_Setup();
+int start_measuring();
+int CountingAlgorithm(int16_t Distance, uint8_t zone);
+int Check_PeopleCount(uint16_t PeopleCount);
+#ifdef debug
+void display_peoplecounter(int peoplecounter);
+void display_zones();
+#endif
+
+// Defines
+#define NOBODY 0

```

Comment:
Kunnen deze defines een duidelijkere naam krijgen? of is een enum hier eventueel netjes?